backend/app/services/knowledge/code_wiki/automatic_update.py

```
@@ -0,0 +1,297 @@
+
+RUNNER_SPEC_KEY = "executionPrincipalUserId"
+INTERNAL_WIKI_KEY = "code_wiki_id"
+
+
+def plan_for(db: Session, knowledge_base_id: int) -> Kind | None:
+    plans = (
+        db.query(Kind)
+        .filter(
+            Kind.kind == "Subscription",
+            Kind.name == f"code-wiki-{knowledge_base_id}",
+            Kind.is_active == True,
```

Ruff (0.16.3)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)